### ISSUE TITLE
Type is not being imported

### ISSUE THREAD

**Issue by OuthBack** [futurGH/ts-to-jsdoc#38]
The output in jsdoc has not the type import.

desired file "example.ts":
```ts
import { ParamType } from "./example2";

function someFunction(param: ParamType) {}
```

file "example2.ts" where the type is located:
```ts
export type ParamType = string;
```

Output JS:
```js
/**
 * @param {ParamType} param
 * @returns {void}
 */
function someFunction(param) { }
export {};
```

**Comment by kungfooman** [comment]
Use proper `import type` - TypeScript removes unused imports.

**Comment by OuthBack** [comment]
In my Typescript files I usually import with just `import`. Is there a way to check if the import is a type?

**Comment by kungfooman** [comment]
> Is there a way to check if the import is a type?

Didn't try it myself, but this seems to be exactly what you ask for, check out this: https://typescript-eslint.io/rules/consistent-type-imports/

**Comment by OuthBack** [comment]
What I meant is if there is a way to create a check if the import is a type in _this_ library(ts-to-jsdoc) to then use it like it is right now.
Example:
input
```ts
import { ParamType } from "./example2";

function someFunction(param: ParamType) {}
```

output
```js
/**
 * @typedef {import('./example2.js').ParamType} ParamType
 */

/**
 * @param {ParamType} param
 * @returns {void}
 */
function someFunction(param) { }
export {};
```

**Comment by kungfooman** [comment]
So you can keep your code where everything is confusing and conflated together? `import type` exists for a reason and you better clean up your code.

**Comment by OuthBack** [comment]
Why the library could not handle that? Not all the code uses `import type`?

**Comment by futurGH** [comment]
You _should_ be using `import type` to import types, but I think it should be possible for `ts-to-jsdoc` to detect when an import is only used in a type position. Will look into that when I get a chance.

**Comment by OuthBack** [comment]
I'm trying to add this feat. But I didn't find a function to get the type of the import yet

**Comment by futurGH** [comment]
Already implemented, just need to write a test or two today and I'll publish ;)

**Comment by futurGH** [comment]
Fixed in 2.0.0!
